scripts/metadata_comparison/metadata_comparison/extractor.py

```diff
@@ -121,13 +88,13 @@ def upload_operations_metadata_json(bucket_name, operation_id, operations_metada
     upload_blob(bucket_name, formatted_metadata, operation_upload_path, gcs_storage_client)
 
 
-def process_workflow(cromwell_url, gcs_bucket, gcs_path, gcs_storage_client, genomics_v2alpha1_client, workflow):
+def process_workflow(cromwell_url, gcs_bucket, gcs_path, gcs_storage_client, papi_clients, workflow):
```

Comment:
One thing that really helps for readability is using type annotation. Currently all versions of python support this. 

Python type annotation is described here: https://docs.python.org/3/library/typing.html .